### PR TITLE
read dat.json metadata and disply title

### PR DESCRIPTION
**Pull request by karissa**

### CODE REVIEW

**Comment by karissa** [comment]
cc @juliangruber 

still isn't doing it live

**Comment by juliangruber** [comment]
@karissa see https://github.com/juliangruber/dat-desktop/pull/41 for a live updating addition

**Comment by juliangruber** [comment]
lgtm!

**Comment by clkao** [comment]
You know we can add an e2e test now for actually testing this!

File: client/js/models/archive.js [hunk 1]
```diff
@@ -159,6 +172,7 @@ module.exports = {
             // XXX: Hack to fetch a small bit of data so size properly updates
           })
         }
+        send('archive:getMetadata', noop)
```
Comment (laurengarcia) [inline client/js/models/archive.js]: am pretty sure this is the wrong signature for `send`, i think you might mean:
`send('archive:getMetadata', {}, noop)`

File: client/js/models/archive.js [hunk 2]
```diff
@@ -84,6 +86,17 @@ module.exports = {
...
       send('archive:load', data, done)
     },
+    getMetadata: function (data, state, send, done) {
+      // EXPERIMENTAL:
+      // right now we are reading this from dat.json but perhaps we
+      // will update this when we start using accounts and repos
+      var archive = state.instance
+      collect(archive.createFileReadStream('dat.json'), (err, raw) => {
+        if (err) return done()
+        const json = JSON.parse(raw.toString())
```
Comment (laurengarcia) [inline client/js/models/archive.js]: what if `raw` is malformed? want to wrap this in try/catch?

**Pull request merged**